dev/clint/src/clint/config.py

```diff
@@ -8,10 +8,13 @@
 @dataclass
 class Config:
     exclude: list[str]
+    # Path -> List of modules that should not be imported globally under that path
+    forbidden_global_imports: dict[str, list[str]]
```

Comment:
```suggestion
    forbidden_top_level_imports: dict[str, list[str]]
```

can we replace `global` with `top_level`?